lib/datadog/kit/appsec/events.rb

```diff
@@ -10,108 +10,128 @@ module Events
         LOGIN_SUCCESS_EVENT = 'users.login.success'
         LOGIN_FAILURE_EVENT = 'users.login.failure'
 
-        # Attach login success event information to the trace
-        #
-        # This method is experimental and may change in the future.
-        #
-        # @param trace [TraceOperation] Trace to attach data to. Defaults to
-        #   active trace.
-        # @param span [SpanOperation] Span to attach data to. Defaults to
-        #   active span on trace. Note that this should be a service entry span.
-        #   When AppSec is enabled, the expected span and trace are automatically
-        #   used as defaults.
-        # @param user [Hash<Symbol, String>] User information to pass to
-        #   Datadog::Kit::Identity.set_user. Must contain at least :id as key.
-        # @param others [Hash<String || Symbol, String>] Additional free-form
-        #   event information to attach to the trace.
-        def self.track_login_success(trace = nil, span = nil, user:, **others)
-          if (appsec_scope = Datadog::AppSec.active_scope)
-            trace = appsec_scope.trace
-            span = appsec_scope.service_entry_span
+        class << self
```

Comment:
Can you split the 78ca839b502d4a467c97dbf95b58f0707773a07c in two?

1. move to `class << self`
2. do the intended change

It's kind of hard to read all mixed up, even when ignoring whitespace changes